cogs/guess.py

```diff
@@ -0,0 +1,91 @@
+"""importsss (that i don't know if it belongs here)
+
+    you can get rid of excess comments"""
+import time
+import random
+from discord.ext import commands
+from random import choice
+from .utils.dataIO import dataIO
+from .utils import checks
+from .utils.chat_formatting import box
+from collections import Counter, defaultdict, namedtuple
+import discord
+import os
+import asyncio
+import chardet
+import re
+"""Guessing game """
+class Guess:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="guess")
+    async def mycom(self):
+        """welcoming the user"""
+        await self.bot.reply("guess what word I'm thinking of!")
+
+        """thinking"""
+        await asyncio.sleep(3)
+
+        """start guess"""
+        await self.bot.say("Start guessing...(hint: no caps/")
+        await asyncio.sleep(0.5)
+
+        # make secret
+        secret = "its gonna change"
+
+        # Probably need to create a word list and grab words from there
+        items = ["here", "are", "some", "strings", "of","which", "we", "will", 
+         "select", "one", "two", "three", "four", "five", "six", 
+         "seven", "eight", "nine", "ten", "python", "ruby", "java", "javascript", "traceback", "print", 
+         "random", "input", "raw", "print", "sleep", "time", "secret", "while", "then", "if", 
+         "loop", "in", "else", "break", "empty", "mocha", "matcha", "money", "spam","jazz","jam","spaz", 
+         "binder", "jokes", "chemistry", "person", "matcha", "night", "everyone", "pasta", "craving", 
+         "jolly", "ranch", "lit", "still", "first", "second", "third", "skull", "ketchup", 
+         "ice", "red", "life", "death", "magic", "comments", "class", "secret", "value", 
+         "variable", "guess", "incorrect", "russian", "roulette", "bot", "turn", 
+         "failed", "wood", "stone", "cobblestone", "blanket", "ore", "chest", "poster"]
+        #picks one word from list
+        secret = items[random.randrange(len(items))]
+
+        #creates a variable with an empty value
+        guesses = ''
+        turns = 5
+        while turns > 0:
+            failed = 0
+            for char in secret:
+                if char in guesses:
+                    await self.bot.say(char),
+                else:
+                    await self.bot.say("_"),
+                    failed += 1
+            """ if failed is equal to zero"""
+            """ Win text"""
+            if failed == 0:        
```

Comment:
This looks like it's being used as a boolean.  Please switch the initialization of this above to `failed = False`, and just check   it with `if failed:`.